### ISSUE TITLE
[Bug] Something wrong with the latency check in latest version in WSL.

### ISSUE THREAD

**Issue by koast18** [nxtrace/NTrace-core#176]
Hi,
I installed the latest version of nexttrace but an error occurs when run `nexttrace 1.1.1.1`, the log is like 
```
NextTrace v1.2.2.2 2023-10-11T02:52:29Z 8b30ef3
[NextTrace API] prefered API IP - [2606:4700:3030::ac43:cd76] - errorms - Get "https://[2606:4700:3030::ac43:cd76]:443/v3/challenge/request_challenge": dial tcp [2606:4700:3030::ac43:cd76]:443: connect: cannot assign requested address
RetToken failed 3 times, please try again after a while, exit
```

The root case maybe the `GetFastIP`  and can be found [here](https://github.com/nxtrace/NTrace-core/blob/main/util/latency.go#L27).

In WSL, it would return an ipv6 address even though there is no available ipv6 address assigned.

Also, I tried the [v1.2.2.0](https://github.com/nxtrace/NTrace-core/releases/tag/v1.2.2.0) and [v1.2.1.1](https://github.com/nxtrace/NTrace-core/releases/tag/v1.2.1.1) in the same env, it just works fine.

**Comment by tsosunchia** [comment]
In order to confirm your problem,

You need to clarify your network type. Is your network an IPv4 single stack or an IPv4/IPv6 dual stack?

And please give feedback on the results of `nslookup api.leo.moe`

**Comment by tsosunchia** [comment]
Guess your question is similar to https://github.com/nxtrace/NTrace-core/issues/175 .

https://github.com/nxtrace/NTrace-V1/commit/960ab9687cff1dd5fb1fc2e7b6b6ca8b640e90c1
You can try the pre-compiled package provided here to see if the problem is solved:
https://github.com/nxtrace/NTrace-V1/actions/runs/6494043746

**Comment by tsosunchia** [comment]
fix with tag v1.2.2.3

**Comment by koast18** [comment]
> fix with tag v1.2.2.3

Tested, it works fine! Thank you.
